src/Core/SharedKernels/Enumerator/Enumerator/ViewModels/InterviewDetails/Questions/CategoricalComboboxAutocompleteViewModel.cs

```diff
@@ -158,29 +200,36 @@ public void ResetFilterAndOptions()
         {
             this.FilterText = null;
             
-            var suggestions = this.GetSuggestions(null).ToList();
+            //var suggestions = this.GetSuggestions(null);
             this.QuestionState.Validity.ExecutedWithoutExceptions();
 
             this.InvokeOnMainThread(() =>
             {
-                this.AutoCompleteSuggestions = suggestions;
+                //this.AutoCompleteSuggestions = suggestions;
+                this.AutoCompleteSuggestions = null;
+                //this.AutoCompleteSuggestions = null;
                 this.RaisePropertyChanged(nameof(this.FilterText));

```

Comment:
make FilterText observable and set it

Reply:
UI control changes FilterText via binding on each change in textbox. If we make this property observable on set, we will have flow: UI textbox ->set property -> raise change event - UI control reread property value.
At current moment we fire change property manually to clear suggestions list.